src/components/Confirm/CaptureViewer/CaptureVideoViewer.js

```diff
@@ -4,7 +4,7 @@ import style from './style.css'
 
 const CaptureVideoViewer = ({ previewUrl }) => (
   <div className={style.videoWrapper}>
-    <video className={style.video} src={previewUrl} controls />
+    <video aria-label="Replay your recorded video" className={style.video} src={previewUrl} controls />
```

Comment:
Same here. This string should be included in the `en.json` locale file and have the relative translation within `es.json`.

Reply:
Done.